base/base_tests/stl_helpers_test.cpp

```diff
@@ -19,95 +20,117 @@ class Int
   int m_v;
 };
 
-UNIT_TEST(LessBy)
+template<template<class, class> class TContainer>
+void TestSortUnique()
```

Comment:
It doesn't work this way. In the test we tests Cont<int>, Cont<pair<int, int>> and Cont<pair<double, int>>. There's a way to split TestSortUnique into 3 tests and call:
```
TestSortUnique<vector<int>>();
TestSortUnique<vector<pair<int, int>>>();
TestSortUnique<vector<pair<double, int>>>();
```